VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitVideoFragment.java

```diff
@@ -281,7 +281,7 @@ public void onPositiveClicked() {
                         }
                         @Override
                         public void onNegativeClicked() {
-                            ((VideoListActivity) getActivity()).showInfoMessage(getString(R.string.wifi_off_message));
+                            ((CourseUnitNavigationActivity) getActivity()).showInfoMessage(getString(R.string.wifi_off_message));
```

Comment:
Since these casts seem easy to get wrong or get screwed up in a refactoring should we do an ``instanceof`` check or something?